uwblueprint/community-fridge-kw
# feat: Add confirmation modal when deleting checkin + removing volunteer

## Brief description. What is this change? 

### [Add confirmation modal when deleting checkin + removing volunteer](https://www.notion.so/uwblueprintexecs/Admin-Checkin-View-Remove-Volunteer-and-Cancel-Shift-0c3642d911174abca4e5e7e60b7c1fdc)



## Implementation description. How did you make this change?
* Used general delete confirmation modal to ask user for confirmation to delete check-in or remove a volunteer



## Steps to test
1. Log in as admin
2. Go to Fridge Check-in view
3. Cancel a check-in (Modal should appear asking user if they want to confirm cancelling it with shift's time and day)
4. Remove an assigned volunteer from a check-in (Modal should appear with shift's time and day)
5. Make sure copy is correct (see ticket for copy)

## Checklist
